packages/vue-app/template/components/nuxt-error.vue

```diff
@@ -1,25 +1,3 @@
-<template>
-  <div class="__nuxt-error-page">
-    <div class="error">
-      <svg xmlns="http://www.w3.org/2000/svg" width="90" height="90" fill="#DBE1EC" viewBox="0 0 48 48">
-        <path d="M22 30h4v4h-4zm0-16h4v12h-4zm1.99-10C12.94 4 4 12.95 4 24s8.94 20 19.99 20S44 35.05 44 24 35.04 4 23.99 4zM24 40c-8.84 0-16-7.16-16-16S15.16 8 24 8s16 7.16 16 16-7.16 16-16 16z" />
-      </svg>
-
-      <div class="title">{{ message }}</div>
-      <p v-if="statusCode === 404" class="description">
-        <NuxtLink class="error-link" to="/"><%= messages.back_to_home %></NuxtLink>
-      </p>
-      <% if(debug) { %>
-      <p class="description" v-else><%= messages.client_error_details %></p>
-      <% } %>
-
-      <div class="logo">
-        <a href="https://nuxtjs.org" target="_blank" rel="noopener"><%= messages.nuxtjs %></a>
-      </div>
-    </div>
-  </div>
-</template>
-
 <script>
```

Comment:
This part is still passed through vue-loader. Maybe we can rename this file to `.js`?

Reply:
Unfortunately yes, when I implemented the changes for nuxt-loading.vue @Atinux mentioned he preferred to keep the js code together with the css in one file (due to maintainability?) so I assumed that'd be the case here as well.

Reply:
Yes. Makes sense for maintainability. So if we still pass it into vue-loader, there are no perf benefits (at least notable) and makes maintenance of error template harder.

Reply:
I agree with @pi0 , since vue-loader will pre-compile the `template` so there'is no harm in runtime and Vue compiler may even have some extra performance enhancements in pre-compiling.